contrib/babelfishpg_tsql/src/multidb.c

```diff
@@ -1131,6 +1137,28 @@ const char *get_guest_role_name(const char *dbname)
 	}
 }
 
+const char *get_guest_schema_name(const char *dbname)
```

Comment:
We cannot completely get rid of `get_guest_schema_name()`. 
This is because `get_physical_schema_name()` errors out when the db doesnt exist

```
else if (!DbidIsValid(get_db_id(db_name)))
		{
			ereport(ERROR,
				(errcode(ERRCODE_UNDEFINED_DATABASE),
					errmsg("database \"%s\" does not exist. Make sure that the name is entered correctly.", db_name)));
		}
```